<?xml version="1.0"?>
<launch>
  <arg name="PlumeType" default="gaussian"/>

  <arg name="plotGaussian" default="True"/>

  <param name="QPlume"                type="double"  value="0.000077" /> #  ppm     release rate
  <param name="vPlume"           type="double"  value="0.447"    /> #  m/s      velocity
  <param name="DyPlume"          type="double"  value="0.001"    /> #  m        diffusion along y
  <param name="DzPlume"          type="double"  value="0.0005"   /> #  m        diffusion along z

  <!-- Plume location -->
  <param name="xPlume"           type="double"  value="0.3048"   /> # m
  <param name="yPlume"           type="double"  value="1.3716"   /> # m
  <param name="zPlume"           type="double"  value="0"        /> # m
  <param name="careAboutZ"       type="bool"    value="True"     /> # boolean
  <param name="thetaPlume"       type="double"  value="0"        /> # degrees (both methods need to know the wind direction)

  <param name="sensorRate"       type="double"  value="1"        /> # Hz

  <include file="$(find mocap_optitrack)/launch/mocap.launch"/>

  <node name="plotGaussianMocap" pkg="platform_sims" type="plotGaussianMocap.py" output="screen" if="$(eval plotGaussian)"/>

  <!-- <node name="gaussianSensorPlatform" pkg="platform_sims" type="gaussianSensorPlatform" output="screen"/> -->

</launch>
